docker-compose.yml

```diff
@@ -5,25 +5,25 @@
 version: '3.3'
 services: 
   webui:
-    image: ${WEBUI_IMAGE}    
+    image: intel/oact-webui:v2.1.1  
```

Comment:
why move it from the .env?

Reply:
We had a convo a while back around 2.1 and said the images in .env felt out of place in the env and belonged in the docker-compose.  I have no preference either way, I can use .env, up to you.